BUILD

```diff
@@ -209,6 +208,11 @@ config_setting(
     values = {"define": "release=true"},
 )
 
+config_setting(
+    name = "static",
```

Comment:
Nit: could this be moved to under `platforms/configs`?

Reply:
It's not really platform-related, though? Like, obviously any config_setting can be in any publicly visible bazel file in any repository, so the answer to your question is yes, but the `static` go build setting is absolutely independent of the platform which makes putting it there kind of odd to me.